test/specs/helpers/buildURL.spec.js

```diff
@@ -17,15 +17,15 @@ describe('helpers::buildURL', function () {
       foo: {
         bar: 'baz'
       }
-    })).toEqual('/foo?foo=' + encodeURI('{"bar":"baz"}'));
+    })).toEqual('/foo?foo=' + encodeURIComponent('{"bar":"baz"}'));
   });
 
   it('should support date params', function () {
     var date = new Date();
 
     expect(buildURL('/foo', {
       date: date
-    })).toEqual('/foo?date=' + date.toISOString());
+    })).toEqual('/foo?date=' + encodeURIComponent(date.toISOString()));
```

Comment:
should this test and line 20 have the actual encoded string we are expecting? since it was hard to tell this was broken if you don't actually run the tests